xbmc/platform/linux/AppParamParserWebOS.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright (C) 2005-2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "AppParamParserWebOS.h"
+
+CAppParamParserWebOS::CAppParamParserWebOS() : CAppParamParserLinux(), m_nArgs(0)
```

Comment:
No need to call the default ctor of the base class. This will be done automatically.